webpack.config.js

```diff
@@ -26,6 +26,10 @@ const OUT_PATH = path.resolve('./build');
 const PUBLIC_PATH = '/assets/';
 const IS_DEV = process.env.MDC_ENV === 'development';
 const IS_PROD = process.env.MDC_ENV === 'production';
+const IS_FAST_BUILD = process.env.MDC_BUILD === 'fast';
+const GENERATE_SOURCE_MAPS = IS_DEV && !IS_FAST_BUILD && process.env.MDC_GENERATE_SOURCE_MAPS !== 'false';
+const WRAP_CSS_IN_JS = process.env.MDC_WRAP_CSS_IN_JS !== 'false';
```

Comment:
Is there a reason this doesn't get flipped by IS_FAST_BUILD?  As with my comment in #1128, I'm confused why we're adding this many separate environment variables.  I had assumed the point of adding IS_FAST_BUILD was to flip *all* the switches to go as fast as possible.